lib/manageiq/appliance_console/database_configuration.rb

```diff
@@ -80,7 +82,8 @@ def create_or_join_region
     end
 
     def create_region
-      ManageIQ::ApplianceConsole::Utilities.bail_if_db_connections("preventing the setup of a database region")
+      hint = "\nPlease stop evm server processes or change their region from all appliances in this region."
```

Comment:
> change their region from all appliances in this region

I don't know what this means. You can't really change an appliances region.

Reply:
Sorry. So the only way to solve this is to stop evm server processes connected to this database?

Reply:
Yes, all of the evmserverd services on appliances in the region should be stopped.